app/src/main/java/com/example/cinemapp/ui/main/MainActivity.kt

```diff
@@ -22,12 +43,158 @@ class MainActivity : AppCompatActivity() {
         setupNavigation()
     }
 
+    override fun onStart() {
+        super.onStart()
+        viewModel.getAccountDetail(false)
```

Comment:
Again getAccuntDetail should ideally not take params for the dialog opened state